README.md

```diff
@@ -251,6 +251,13 @@ The following types/features are not supported for generated models:
   another type, is supported.
 - Reused models for nested records: Currently an anonymous model class is
   generated for each subrecord.
+  
+### Test Utilities
+
+Tests that leverage the Messaging API for encoding/decoding will need to talk to schema registry. 
```

Comment:
After offline discussion we're going to remove the README change and leave this undocumented for now.